File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {

        int longest = 0;
        unordered_set<int> numSet(nums.begin(), nums.end());
        for(int num : nums) {
            if(!numSet.count(num - 1 )) {
                int currentNum = num;
                int streak = 1;
                while(numSet.count(currentNum + 1)) {
                    currentNum++;
                    streak++;

                }
                longest = max(longest, streak);
            }
        }

        return longest;
        
    }

};

File: Data Structures & Algorithms/two-integer-sum/submission-0.go
func twoSum(nums []int, target int) []int {

    length := len(nums)

    solution := make([]int, 2)

    for i:= 0; i < length; i++ {
        for j:= i + 1; j < length; j ++ {
            if nums[i]+nums[j] == target {
                solution[0] = i
                solution[1] = j
                return solution
            }
        }
    }





    return nil
}

